src/main/java/org/sonar/plugins/jacoco/JacocoSensor.java

```diff
@@ -58,7 +58,11 @@ void importReports(ReportPathsProvider reportPathsProvider, FileLocator locator,
         LOG.warn("Report doesn't exist: '{}'", reportPath);
       } else {
         LOG.debug("Reading report '{}'", reportPath);
-        importReport(new XmlReportParser(reportPath), locator, importer);
+        try {
+          importReport(new XmlReportParser(reportPath), locator, importer);
+        } catch (Exception e) {
+          LOG.error("Coverage report '{}' could not be read/imported. Error: {}", reportPath, e);
```

Comment:
If there is a problem with the `importCoverage`, it will throw 2 exceptions nested. Maybe this could be improved. If the constructor of `XmlReportParser` doesn't throw an exception, maybe we can move this catch inside `importReport` where we call `reportParser.parse`.